node/src/actors/chain_manager/mod.rs

```diff
@@ -1207,13 +1212,21 @@ impl ChainManager {
             .map_err(|e, _, _| log::error!("Superblock building failed: {:?}", e))
             .and_then(move |(block_headers, last_hash), act, ctx| {
                 let consensus = if act.sm_state == StateMachine::Synced || act.sm_state == StateMachine::AlmostSynced {
-                    if voted_superblock_beacon.checkpoint + 1 != superblock_index {
+
+                    if voted_superblock_beacon.checkpoint == last_consolidated_beacon.checkpoint {
+                        log::debug!("Counting votes for a already consolidated superblock index {}", superblock_index);
+                        SuperBlockConsensus::SameAsLocal
+                    }
+                    else if voted_superblock_beacon.checkpoint + 1 != superblock_index {
                         // Warn when there is are missing superblocks between the one that will be
                         // consolidated and the one that will be created
                         log::warn!("Counting votes for Superblock {:?} when the current superblock index is {}", voted_superblock_beacon, superblock_index);
+                        act.chain_state.superblock_state.has_consensus()
+                    }
+                    else{
```

Comment:
cargo fmt didn't work here

```suggestion
                    } else {
```